neo/IO/Data/LevelDB/Helper.cs

```diff
@@ -11,33 +11,32 @@ public static void Delete(this WriteBatch batch, byte prefix, ISerializable key)
             batch.Delete(SliceBuilder.Begin(prefix).Add(key));
         }
 
-        public static IEnumerable<T> Find<T>(this DB db, ReadOptions options, byte prefix) where T : class, ISerializable, new()
+        public static IEnumerable<T> Find<T>(this LevelDBCore db, ReadOptions options, byte prefix) where T : class, ISerializable, new()
         {
-            return Find(db, options, SliceBuilder.Begin(prefix), (k, v) => v.ToArray().AsSerializable<T>());
+            return Find(db, options, SliceBuilder.Begin(prefix), (k, v) => v.AsSerializable<T>());
         }
 
-        public static IEnumerable<T> Find<T>(this DB db, ReadOptions options, Slice prefix, Func<Slice, Slice, T> resultSelector)
+        public static IEnumerable<T> Find<T>(this LevelDBCore db, ReadOptions options, Slice prefix, Func<byte[], byte[], T> resultSelector)
```

Comment:
Why change this?

Reply:
@vncoelho asked for rename this classes

Reply:
I mean why change `Func<Slice, Slice, T>` to `Func<byte[], byte[], T>`?